apps/web/hooks/escrow/useEscrow.ts

```
@@ -0,0 +1,211 @@
+            setRawRecord(newRecord)
+            const transformed = transformRecord(newRecord)
+            if (transformed) setStatus(transformed)
+          }
+        },
+      )
+      .subscribe()
+
+    return () => {
+      channel.unsubscribe()
+    }
+  }, [escrowId, fetchEscrowStatus, supabase, transformRecord])
```

Biome (1.9.4)

[error] fetchEscrowStatus changes on every re-render and should not be used as a hook dependency.

To fix this, wrap the definition of fetchEscrowStatus in its own useCallback() hook.

(lint/correctness/useExhaustiveDependencies)